eno-core/src/main/java/fr/insee/eno/core/model/navigation/ComponentFilter.java

```diff
@@ -39,14 +39,17 @@ public class ComponentFilter extends EnoObject {
     @Getter
     private final Set<BindingReference> bindingReferences = new HashSet<>();
 
+    @Getter
+    private final Set<BindingReference> resolvedBindingReferences = new HashSet<>();
```

Comment:
This attribute is introduced to make a proper separation between "unresolved" vs "resolved" dependencies? (i.e. to have less mutations) If so, I think the same idea should be applied in the `CalculatedExpression` object which has the same pattern (initial bindigReferences set which is updated ("resolved") in a processing)

Reply:
\+ maybe a line of javadoc on this attribute (or both attributes) would be helpful

Reply:
You're right, we need unresolved dependencies in filter result calculated variables, so i introduced the resolved depencies attributes for condition filter. I'm not sure about introducting the same pattern to CalculatedExpression, as unresolved dependencies are not needed. Maybe we should rename the attribute as resolvedDependencies instead ?